Signal/src/ViewControllers/ProfileViewController.m

```diff
@@ -577,6 +577,7 @@ - (NSString *)avatarActionSheetTitle
 
 - (void)avatarDidChange:(UIImage *)image
 {
+    OWSAssert([NSThread isMainThread]);
```

Comment:
I fixed an unrelated concurrency issue in the avatar helper while testing these changes.